specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -8318,6 +8952,253 @@
+        }
+      },
+      "description": "capacity reservation Group Properties."
+    },
+    "CapacityReservationGroup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CapacityReservationGroupProperties"
+        },
+        "zones": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Availability Zones to use for this capacity reservation group. The zones can be assigned only during creation. If not provided, the group supports only regional resources in the region. If provided, enforces each capacity reservation in the group to be in one of the zones."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Specifies information about the capacity reservation group that the capacity reservations should be assigned to. <br><br> Currently, a capacity reservation can only be added to a capacity reservation group at creation time. An existing capacity reservation cannot be added to another capacity reservation group."
+    },
+    "CapacityReservationGroupUpdate": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CapacityReservationGroupProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/UpdateResource"
+        }
+      ],
+      "description": "Specifies information about the capacity reservation group that the capacity reservation should be assigned to. Only tags can be updated."
+    },
+    "CapacityReservationGroupListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CapacityReservationGroup"
+          },
+          "description": "The list of capacity reservation groups"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of capacity reservation Groups. Call ListNext() with this URI to fetch the next page of capacity reservation Groups."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The List capacity reservation Group with resource group response."
+    },
+    "CapacityReservationInstanceView": {
+      "type": "object",
+      "properties": {
+        "utilizationInfo": {
+          "$ref": "#/definitions/CapacityReservationUtilization",
+          "description": "Unutilized capacity of the capacity reservation."
+        },
+        "statuses": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstanceViewStatus"
+          },
+          "description": "The resource status information."
+        }
+      },
+      "description": "The instance view of a capacity reservation."
```

Comment:
update to description

that provides as snapshot of the runtime properties of the capacity reservation that is managed by the platform and can change outside of control plane operations.

Reply:
updated